spec/components/candidate_interface/additional_referees_start_component_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::AdditionalRefereesStartComponent do
+  let(:application_form) { create(:completed_application_form, references_count: 2, with_gces: true) }
+  let(:first_referee) { application_form.application_references.first }
+  let(:second_referee) { application_form.application_references.second }
+
+  context 'when one referee request failed' do
+    it 'has a page content that requests one new referee' do
+      first_referee.update!(feedback_status: 'email_bounced')
```

Comment:
You could get rid of the `let`s by creating the required references in the `it` block itself - perhaps cleaner?

```suggestion
      create(:reference, :bounced, application_form: application_form) 
```

Reply:
I don't think it makes it cleaner, but I had a go: 2636b37

Reply:
I think in this cases those `let` blocks are acceptable, I need to assign the references to a var anyway because they are referenced in the assertions.  I let you comment on this refactoring, but I would prefer to keep these tests as they are 😬